source/framefmt.c

```diff
@@ -228,8 +240,44 @@ bool aws_cryptosdk_frame_is_valid(const struct aws_cryptosdk_frame *const frame)
     bool ciphertext_valid  = ciphertext_byte_buf_valid || ciphertext_valid_zero;
     bool ciphertext_static = frame->ciphertext.allocator == NULL;
 
-    return iv_byte_buf_valid && iv_byte_buf_static && authtag_byte_buf_valid && authtag_byte_buf_static &&
-           ciphertext_valid && ciphertext_static;
+    return (
+        frame_type_valid && iv_byte_buf_valid && iv_byte_buf_static && authtag_byte_buf_valid &&
+        authtag_byte_buf_static && ciphertext_valid && ciphertext_static);
+}
```

Comment:
```c
return (authtag_byte_buf_static &&
        authtag_byte_buf_valid &&
        ciphertext_static &&
        ciphertext_valid &&
        frame_type_valid &&
        iv_byte_buf_static &&
        iv_byte_buf_valid);
```

Reply:
Apply the same pattern for the other functions.

Reply:
For any comments like this, we should just run ./reformat.sh, and accept whatever it does.

Reply:
Right!! The important thing is just to follow the `reformat.sh`.